.appveyor/build_libedit.bat

```diff
@@ -66,7 +66,9 @@ if not exist "%BINARIESDIR%\wineditline-%LIBEDIT_VERSION%.zip" (
 )
 
 2>nul rd /S /Q "%BINARIESDIR%\wineditline-%LIBEDIT_VERSION%"
-7z x -y "%BINARIESDIR%\wineditline-%LIBEDIT_VERSION%.zip" -o"%BINARIESDIR%"
+
+REM Use the ENV_7Z_BIN environment variable
```

Comment:
@cherrysx We don't have to add comments at every use of ENV_7Z_BIN. (At least for me) it's implicit that we if define any variable in set_paths.bat, we have to use it everywhere in *.bat files.